File: prog/c/fibonaccis.c
// fibonaccis.c - print all Fibonacci numbers < 1 << 64

#include <inttypes.h>
#include <stdint.h>
#include <stdio.h>

#define N 94

int main(void) {
	uint64_t fs[N];

	fs[0] = 0;
	fs[1] = 1;

	for (uint8_t n = 2; n < N; ++n)
		printf("%" PRIu64 "\n", fs[n] = fs[n - 1] + fs[n - 2]);
}

File: prog/python/fibonaccis.py
#! /usr/bin/env python3

# fibonaccis.py - print all Fibonacci numbers < 2 ** 64

fs = [0, 1]

print(fs[0])
print(fs[1])

while True:
	fs.append(fs[-1] + fs[-2])

	if fs[-1] < 2 ** 64:
		print(fs[-1])
	else:
		break
